scribus/plugins/import/pdf/slaoutput.cpp

```diff
@@ -2509,11 +2513,11 @@ void SlaOutputDev::drawSoftMaskedImage(GfxState *state, Object *ref, Stream *str
 	ImageStream * imgStr = new ImageStream(str, width, colorMap->getNumPixelComps(), colorMap->getBits());
 	imgStr->reset();
 	unsigned int *dest = nullptr;
-	unsigned char * buffer = new unsigned char[width * height * 4];
+	unsigned char * buffer = new unsigned char[(long)width * (long)height * 4];

```

Comment:
i'm not sure that these changes belong to this patch...

if they do not, i think you should submit it as a separate patch (i guess that you can submit them already)